backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -16,26 +16,27 @@ RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
 # Python checks
-PYTHON?=python3
+UV?=uv
 
 # installed?
-ifeq (, $(shell which $(PYTHON) ))
-  $(error "PYTHON=$(PYTHON) not found in $(PATH)")
-endif
-
-# version ok?
-PYTHON_VERSION_MIN=3.10
-PYTHON_VERSION_OK=$(shell $(PYTHON) -c "import sys; print((int(sys.version_info[0]), int(sys.version_info[1])) >= tuple(map(int, '$(PYTHON_VERSION_MIN)'.split('.'))))")
-ifeq ($(PYTHON_VERSION_OK),0)
-  $(error "Need python $(PYTHON_VERSION) >= $(PYTHON_VERSION_MIN)")
+ifeq (, $(shell which $(UV) ))
+  $(error "UV=$(UV) not found in $(PATH)")
 endif
 
 BACKEND_FOLDER=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
-GIT_FOLDER=$(BACKEND_FOLDER)/.git
+ifdef PLONE_VERSION
+PLONE_VERSION := $(PLONE_VERSION)
+else
+PLONE_VERSION := {{ cookiecutter.plone_version }}
+endif
+
 VENV_FOLDER=$(BACKEND_FOLDER)/.venv
 BIN_FOLDER=$(VENV_FOLDER)/bin
 
+# Environment variables to be exported
+export PYTHONWARNINGS := ignore
```

Comment:
I think this is a really bad default.

Reply:
I see it was there before, which I didn't realize, and this is just moving it. Meh.

Reply:
I would like to see warnings to help me identify and prepare for deprecations.

Reply:
To be honest, I always add that because of the amount of `plone.base` warnings during pytest runs.